app/controllers/devise/two_step_verification_controller.rb

```diff
@@ -27,7 +27,7 @@ def update
   end
 
   def otp_secret_key_uri
-    issuer = "GOV.UK%20Signon"
+    issuer = I18n.t('devise.issuer')
```

Comment:
@jamiecobbett  you are correct - had done that in the locale file. This has been fixed in a separate branch https://github.com/alphagov/signonotron2/pull/471